enterprise/frontend/src/embedding-sdk/components/private/AppInitializeController.tsx

```diff
@@ -23,14 +26,33 @@ export const AppInitializeController = ({
   useInitData({ config });
 
   const isInitialized = useSdkSelector(getIsInitialized);
+  const licenseProblem = useSdkLicenseProblem(config);
+
+  const content = useMemo(() => {
+    if (!isInitialized) {
+      return <SdkLoader />;
+    }
+
+    if (licenseProblem?.severity === "error") {
+      return null;
+    }
+
+    return children;
+  }, [children, isInitialized, licenseProblem]);
 
   return (
     <SdkGlobalStylesWrapper
       baseUrl={config.metabaseInstanceUrl}
       id={EMBEDDING_SDK_ROOT_ELEMENT_ID}
       className={className}
     >
-      {!isInitialized ? <div>{t`Loading…`}</div> : children}
+      {content}
+
+      {licenseProblem && (
+        <Box pos="fixed" bottom="15px" left="15px">
```

Comment:
The indicator floats on the bottom-left corner.

![CleanShot 2567-09-11 at 05 24 12@2x](https://github.com/user-attachments/assets/467cf432-c7d8-4dab-acb0-a5d04a12fd50)